kratos/geometries/geometry.h

```diff
@@ -1375,7 +1375,7 @@ class Geometry : public PointerVector<TPointType>
         for ( unsigned int pnt = 0; pnt < this->IntegrationPointsNumber( ThisMethod ); pnt++ )
         {
             this->Jacobian( J, pnt, ThisMethod);
-            rResult[pnt] = MathUtils<double>::Det(J);
```

Comment:
generalized det is more costly than normal determinant, so you can not substitute it in the base class geometry.
a better solution if we reach an agreement about this is to reimplement the function in the derived classes of the geometry. However please do not do it right now, we are still discussing it in #80